src/InputWithLabel.jsx

```diff
@@ -0,0 +1,23 @@
+import { useRef, useEffect } from 'react'
+
+function InputWithLabel({ children, value, onChange, id }) {
+    const inputRef = useRef(null);
+  
+    useEffect(() => {
+      inputRef.current.focus();
+    }, []);
```

Comment:
@A-Kaliexe take out the dependency array from the `useEffect` so that the input is auto focused on every re-render as opposed to only once on mount